pyteal/ast/app_test.py

```diff
@@ -162,94 +162,302 @@ def test_local_put():
     args = [Int(0), Bytes("key"), Int(5)]
     expr = App.localPut(args[0], args[1], args[2])
     assert expr.type_of() == TealType.none
-    
+
     expected = TealSimpleBlock([
         TealOp(args[0], Op.int, 0),
         TealOp(args[1], Op.byte, "\"key\""),
         TealOp(args[2], Op.int, 5),
         TealOp(expr, Op.app_local_put)
     ])
-    
+
     actual, _ = expr.__teal__(options)
     actual.addIncoming()
     actual = TealBlock.NormalizeBlocks(actual)
-    
+
     assert actual == expected
 
 def test_local_put_invalid():
     with pytest.raises(TealTypeError):
         App.localPut(Txn.sender(), Bytes("key"), Int(5))
-    
+
     with pytest.raises(TealTypeError):
         App.localPut(Int(1), Int(0), Int(5))
-    
+
     with pytest.raises(TealTypeError):
         App.localPut(Int(1), Bytes("key"), Pop(Int(1)))
 
 def test_global_put():
     args = [Bytes("key"), Int(5)]
     expr = App.globalPut(args[0], args[1])
     assert expr.type_of() == TealType.none
-    
+
     expected = TealSimpleBlock([
         TealOp(args[0], Op.byte, "\"key\""),
         TealOp(args[1], Op.int, 5),
         TealOp(expr, Op.app_global_put)
     ])
-    
+
     actual, _ = expr.__teal__(options)
     actual.addIncoming()
     actual = TealBlock.NormalizeBlocks(actual)
-    
+
     assert actual == expected
 
 def test_global_put_invalid():
     with pytest.raises(TealTypeError):
         App.globalPut(Int(0), Int(5))
-    
+
     with pytest.raises(TealTypeError):
         App.globalPut(Bytes("key"), Pop(Int(1)))
 
 def test_local_del():
     args = [Int(0), Bytes("key")]
     expr = App.localDel(args[0], args[1])
     assert expr.type_of() == TealType.none
-    
+
     expected = TealSimpleBlock([
         TealOp(args[0], Op.int, 0),
         TealOp(args[1], Op.byte, "\"key\""),
         TealOp(expr, Op.app_local_del)
     ])
-    
+
     actual, _ = expr.__teal__(options)
     actual.addIncoming()
     actual = TealBlock.NormalizeBlocks(actual)
-    
+
     assert actual == expected
 
 def test_local_del_invalid():
     with pytest.raises(TealTypeError):
         App.localDel(Txn.sender(), Bytes("key"))
-    
+
     with pytest.raises(TealTypeError):
         App.localDel(Int(1), Int(2))
 
 def test_global_del():
     arg = Bytes("key")
     expr = App.globalDel(arg)
     assert expr.type_of() == TealType.none
-    
+
     expected = TealSimpleBlock([
         TealOp(arg, Op.byte, "\"key\""),
         TealOp(expr, Op.app_global_del)
     ])
-    
+
     actual, _ = expr.__teal__(options)
     actual.addIncoming()
     actual = TealBlock.NormalizeBlocks(actual)
-    
+
     assert actual == expected
 
 def test_global_del_invalid():
     with pytest.raises(TealTypeError):
         App.globalDel(Int(2))
+
+
```

Comment:
nit: consider using single line breaks here for consistency with the above tests